source/quark/R/01_QuarkData.R

```diff
@@ -158,7 +161,10 @@ QuarkData$methods(
         nImps      =  0L,
         compFormat = "",
         miDatasets = NULL,
-        miceObject = NULL
+        miceObject = NULL,
+        #Additional field initialization
+        useParallel = FALSE,
```

Comment:
The "useParallel" and "nProcess" arguments are never used below, so these fields are never populated in the initialized QuarkData object. You need to include "useParallel <<- useParallel" and "nProcess <<- nProcess" in the body of the initializer function.